docs/docsource/operation/XDM_GUI/location_management/adding_a_storage_location.rst

```diff
@@ -0,0 +1,278 @@
+.. _orbit_add_location:
+
+Adding a Storage Location
+=========================
+
+To add a storage location:
+
+#. From the Account view, select an account name to expose account information.
+
+   .. image:: ../../Graphics/xdm_ui_account_view.png
+      :align: center
+	      
+#. Click the **Locations** tab.
+
+   .. image:: ../../Graphics/xdm_ui_locations_tab.png
+      :align: center
+	      
+#. Click **Create Location**.
+
+#. The **ADD NEW STORAGE LOCATION** dialog displays:
+
+   .. image:: ../../Graphics/xdm_ui_add_new_storage_location.png
+      :align: center
+
+   a. Enter a location name in the **Location Name** field using
+      lowercase letters, numbers, and dashes.
+
+      .. note::
+
+         Capital letters, spaces, and punctuation and diacritical
+         marks will result in an error message.
+
+   b. Select a location type from the **Location Type** pull-down menu.
+      You can choose:
+
+      * Amazon S3
+      * Ceph RADOS Gateway
+      * DigitalOcean Spaces
+      * Google Cloud Storage
+      * Microsoft Azure Blob Storage
+      * NFS Mount
+      * Scality RING with S3 Connector
+      * Scality RING with sproxyd Connector
+      * Scality XCore
+      * Wasabi
+      * A Zenko local filesystem
+
+#. Each storage location type has its own requirements. These requirements are
+   detailed in :ref:`Cloud Storage Locations`. No security is required for the
+   local filesystem, but all public clouds require authentication information.
+
+   .. note::
+
+      Adding a location requires credentials (an access key and a secret key).
+      Though nothing prevents you from using account-level credentials when
+      Zenko requests credentials for a location, it is a best practice to enter
+      credentials specifically generated for this access. In other words, before
+      you add a location, first create a user in that location (an AWS or S3
+      Connector account, for example) for the purpose of |product| access. Give
+      that user all and only the permissions needed to perform the desired
+      tasks.
+
+   .. tip::
+   
+      When configuring an S3 Connector, assign the following policy to the
+      special |product|-access user to ensure access to the Metadata service and the
+      ability to perform operations on the bucket:
+
+      ::
```

Comment:
I would add the json key here for syntax highlighting. I think its like `..codeblock:: json` or something.

Reply:
done